desktop/cmp/form/FormField.js

```diff
@@ -59,7 +59,9 @@ export const [FormField, formField] = hoistCmp.withFactory({
             notValid = model?.isNotValid || false,
             displayNotValid = validationDisplayed && notValid,
             errors = model?.errors || [],
-            requiredStr = (isRequired && !readonly) ? span(' *') : null;
+            requiredIndicator = (isRequired && !readonly) ?
+                defaultProp('requiredIndicator', props, formContext, ' *') :
```

Comment:
+ need the space to be applied to any app applied string.  App should not have to worry about that.

+ More Importantly, see Anselm's point implementing the overrideable styling.